src/defaults.ini

```diff
@@ -1,15 +0,0 @@
-
```

Comment:
This file has been there since the beginning and we've never used it -- we've gone in a different direction for our config files. 

So it's gone! 💨